CondFormats/GeometryObjects/interface/PTrackerParameters.h

```diff
@@ -115,6 +115,8 @@ class PTrackerParameters
   TIBItem tib;
   TIDItem tid;
   TOBItem tob;
+
+  std::vector<int> pDetIdShifts;
```

Comment:
@ianna @davidlange6 I checked that it is not yet in production, so it is still ok to do the change.
I let @ggovi comment further.